groups/infrastructure/module-albs/file-transfer-secure.tf

```diff
@@ -0,0 +1,119 @@
+resource "aws_lb" "secure_file_transfer_alb" {
+  count = "${var.secure_file_transfer_create_alb == 1 ? 1 : 0}"
+
+  name               = "${var.environment}-secure-file-transfer"
+  subnets            = var.subnet_ids
+  security_groups    = ["${aws_security_group.secure-file-transfer_sg.id}"]
+  internal           = "${var.secure_file_transfer_internet_facing == 0 ? true : false}"
+  load_balancer_type = "application"
+  idle_timeout       = 400
+
+  tags ={
+    Environment = "${var.environment}"
+    Name        = "${var.environment}-secure-file-transfer"
+    ALB         = "true"
+  }
+}
+
+resource "aws_lb_listener" "secure_file_transfer_80" {
+  count = "${var.secure_file_transfer_create_alb == 1 ? 1 : 0}"
+
+  load_balancer_arn = "${aws_lb.secure_file_transfer_alb[0].arn}"
+  port              = "80"
+  protocol          = "HTTP"
+
+  default_action {
+    type             = "forward"
+    target_group_arn = "${aws_lb_target_group.secure_file_transfer_8080[0].arn}"
+  }
+}
```

Comment:
I don't think we want to support http routing in this lb and also don't think we should point it to Mesos if we did